server.go

```diff
@@ -48,26 +38,47 @@ type (
 	Locals map[string]interface{} // passed to templates
 )
 
+func defaultMiddlewares() []func(http.Handler) http.Handler {
+	crsfOpts := []csrf.Option{
+		csrf.Path("/"),
+		csrf.FieldName("csrf"),
+		csrf.CookieName(xCSRF_COOKIE_NAME),
+	}
+
+	if serveInsecure == true {
+		crsfOpts = append(crsfOpts, csrf.Secure(false))
+	}
+
+	middlewares := []func(http.Handler) http.Handler{
+		methodOverrideHandler,
+		csrf.Protect([]byte(os.Getenv("SESSION_SECRET")), crsfOpts...),
+		requestLoggerHandler,
+	}
+
+	return middlewares
```

Comment:
This can be simplified to use `!serveInsecure` directly as parameter for `csrf.Secure` and in this case we don't need `if` and we don't need to split csrfOpts before `middlewares`
```suggestion
	return []func(http.Handler) http.Handler{
		methodOverrideHandler,
		csrf.Protect([]byte(os.Getenv("SESSION_SECRET")), []csrf.Option{
			csrf.Path("/"),
			csrf.FieldName("csrf"),
			csrf.CookieName(xCSRF_COOKIE_NAME),
			csrf.Secure(!serveInsecure),
		},
		),
		requestLoggerHandler,
	}
```